queries.sql

```diff
@@ -0,0 +1,8 @@
+--Index medical history foreign key
+CREATE INDEX medical_history_asc ON invoices(medical_history_id ASC);
+
+--Index invoice_id foreign key
+CREATE INDEX invoice_id_asc ON invoice_items(invoice_id ASC);
+
+--Index treatment_id forein key
+CREATE INDEX treatment_id_asc ON invoice_items(treatment_id ASC);
```

Comment:
- [ ] Well done indexing your Foreign Keys! I would suggest indexing all of them (6) as required by the activity.